src/hash-table.js

```diff
@@ -7,6 +7,52 @@ class HashTable {
     this.storage = new LimitedArray(this.limit);
     // Do not modify anything inside of the constructor
   }
+
+  insert(key, value) {
+    if (this.storage.length / this.limit > 0.75) this.limit *= 2;
+    const index = getIndexBelowMax(`${key}`, this.limit);
+    const bucket = this.storage.get(index);
+    const keyVal = [`${key}`, value];
```

Comment:
No need to ensure that the key is a string in this case, since arrays can handle all sorts of types.